item-search/README.md

```diff
@@ -75,6 +49,8 @@ GET /search?collections=landsat8,sentinel&bbox=-10.415,36.066,3.779,44.213&limit
 }
 ```
 
+For more examples see [examples.md](examples.md).
```

Comment:
Maybe also put the link into the "intro" list, here people may miss it.

Reply:
what's the 'intro' list?

Reply:
The list with maturity, conformance uri etc.

Reply:
line 3-6 in the file